tempto-examples/docker/docker-compose.yml

```diff
@@ -30,6 +30,13 @@ services:
       POSTGRES_USER: blah
       POSTGRES_PASSWORD: blah
 
+  cassandra:
```

Comment:
Ouch, sorry. I didn't get your point here. No, we don't need to add it to the `run_on_docker.sh` script. It calls `${DOCKER_COMPOSE} up -d` which starts all dockers in the same compose.